packages/angular/src/generators/application/__snapshots__/application.spec.ts.snap

```diff
@@ -623,9 +628,12 @@ exports[`app --project-name-and-root-format=derived should generate correctly wh
   },
   "extends": "../../tsconfig.base.json",
   "include": [
-    "src/**/*.ts",
-    "src/**/*.js",
+    "**/*.ts",
+    "**/*.js",
     "cypress.config.ts",
+    "**/*.cy.ts",
+    "**/*.cy.js",
+    "**/*.d.ts",
```

Comment:
Wouldn't `**/*.ts` already include `**/*.cy.ts` and `**/*.d.ts` files? CC: @barbados-clemens do you know why it was set up this way?

Reply:
Actually, I understand now. The component tests could be nested in `src` (default), so we're scoping to include those in addition to the project sources.